clang/lib/DPCT/APINamesMemory.inc

```diff
@@ -613,6 +613,23 @@ ASSIGNABLE_FACTORY(FEATURE_REQUEST_FACTORY(
                             ARG(0), ARG(1), ARG(2), ARG(3), ARG(4),
                             DEREF(makeCallArgCreatorWithCall(5))))))
 
+CONDITIONAL_FACTORY_ENTRY(
+    UsePeerAccess(),
+    ASSIGNABLE_FACTORY(ASSIGN_FACTORY_ENTRY(
+        "cuDeviceCanAccessPeer", DEREF(ARG_WC(0)),
+        MEMBER_CALL(CALL(MapNames::getDpctNamespace() +
+                             "dev_mgr::instance().get_device",
+                         ARG_WC(1)),
+                    false, "ext_oneapi_can_access_peer",
+                    CALL(MapNames::getDpctNamespace() +
```

Comment:
this should be another member call

Reply:
We only want to get the second device and pass it to `ext_oneapi_can_access_peer`. And we don't want to access any members from the device returned from get_device().

Therefore, it's just a call.

Reply:
"dpct::dev_mgr::instance().get_device(device).ext_oneapi_can_access_peer(dpct::dev_mgr::instance().get_device(peerDevice));"

The first dpct::dev_mgr::instance().get_device(device) is a member call.
ext_oneapi_can_access_peer() is another membercall.
The arg "dpct::dev_mgr::instance().get_device(peerDevice)" is also a member call.

Reply:
Replaced CALL with MEMBER_CALL template.